tox.ini

```diff
@@ -11,4 +11,4 @@ deps =
     -rtest-requirements.txt
 
 commands =
-    pylint: pylint graph_playbook.py
+    pylint: pylint src/ansible_repo_grapher/graph_playbook.py
```

Comment:
I had not added `graph_folder.py` to linting yet because I had taken care of Python3 issues.  With your changes does it pass checks and can be added here?  If not, I can work on fixing up `graph_folder.py` separately.